test/verify/check-metrics

```diff
@@ -1459,9 +1459,9 @@ class TestMultiCPU(testlib.MachineCase):
 class TestGrafanaClient(testlib.MachineCase):
 
     provision = {
-        "0": {"address": "10.111.112.1/20", "dns": "10.111.112.1", "memory_mb": 512},
+        "0": {"address": "10.111.112.1/20", "dns": "10.111.112.1", "memory_mb": 600},
         # forward Grafana port, so that a developer can connect to it with local browser
-        "services": {"image": "services", "forward": {"3000": 3000}, "memory_mb": 512}
+        "services": {"image": "services", "forward": {"3000": 3000}, "memory_mb": 424}
```

Comment:
I'm actually surprised that TestGrafanaClient passed 3x everywhere with that little RAM -- all the flakes are in other tests. I had expected the services image to need more, not less :grin: But so much the better.

This is a "cost 2" test, i.e. it has the right to allocate ~ 2 GiB, so it does not actually have to be *that* stingy. To avoid future trouble, I'd recommend to give 1024 to "0" and 768 to "services".

Reply:
I wasn't aware of the 2GiB total limit in that case, yes lets bump it. I have a suspicion there is a memory leak in RHEL-10-1 though seeing also the check-networkmanager-mac test fail with a journal issue.